### PR TITLE
Changed language Drop-Down position

### PR DESCRIPTION
**Pull request by gabru-md**
Changes the position of language drop-down, placing it on the bottom-left of the screen.

Closes https://github.com/coala/projects/issues/363

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yukiisbored** [comment]
@gabru-md How does this look on mobile devices and how does it compare to desktop? Screenshots would be nice :)

**Comment by gabru-md** [comment]
@yukiisbored  I've added the relevant screen shots. Please have a look :)
Thanks

s1
The above image is for Desktop Browser

s2
The above image is for Mobile Browser

**Comment by yukiisbored** [comment]
@gabru-md is there a way to fix the one on mobile so it'll be always on the left side.

**Comment by gabru-md** [comment]
yes, let me look through it.
:+1:

**Comment by gabru-md** [comment]
Changed Mobile View : 
m1
Changed Browser View :
w1
I've made them into center. looks better and the same on both platforms :D
@yukiisbored please review :D

File: index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
       </div>
       <div ng-view=""></div>
     </div>
-    <div ng-controller="LanguageController as langCtrl" style="position: fixed; left: 60px; bottom: 20px;">
+    <div ng-controller="LanguageController as langCtrl" style="position: relative; margin:50% auto;width:100px">
```
Comment (hemangsk) [inline index.html]: sorry for nitpicking, but it'll be better to use consistent spacing  
```
position: relative; margin:50% auto;width:100px
```
to
``` 
position: relative; margin: 50% auto; width: 100px;
```

File: index.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
       </div>
```
Comment (hemangsk) [inline index.html]: File name missing in commit message.

**Comment by hemangsk** [comment]
Added a few comments. Screenshots look great! nice work :) 👍

**Comment by gabru-md** [comment]
Okay ! I'll do it :D

**Comment by gabru-md** [comment]
@hemangsk please have a look!
thanks

**Comment by hemangsk** [comment]
Hey @gabru-md , your commit short log doesn't match the commit guidelines
http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog says 
it should have a tag and a short description separated by a colon (:).

**Comment by yukiisbored** [comment]
Please squash your commits.

**Comment by gabru-md** [comment]
@yukiisbored Please Review 
I've squashed all my commits into one.

File: index.html [hunk 3]
```diff
@@ -44,7 +44,7 @@
...
       <div ng-view=""></div>
     </div>
-    <div ng-controller="LanguageController as langCtrl" style="position: fixed; left: 60px; bottom: 20px;">
+    <div ng-controller="LanguageController as langCtrl" style="position: relative; margin: 50% auto; width: 100px">
```
Comment (stellargo) [inline index.html]: semicolon needed after 100px i guess :P

**Comment by soumya-ranjan7** [comment]
ack 8d376be

**Comment by hemangsk** [comment]
@rultor merge

**Comment by hemangsk** [comment]
@gitmate-bot rebase

**Comment by hemangsk** [comment]
ack 1fb3903

**Comment by yukiisbored** [comment]
@rultor merge

**Pull request merged**